docs/get-started/v2/get-started/features/recordings/live-stream-recording.mdx

```diff
@@ -0,0 +1,143 @@
+| Default Template Configuration (At template creation) | Unzipped M3U8                                                                                       | Unzipped M3U8                                                                    |
+| Zipped Recording (M3U8) Upload                        | Not Available                                                                                       | Available                                                                        |
+| Unzipped Recording (M3U8) Upload                      | Available                                                                                           | Available                                                                        |
+| Direct VOD Playback (Via CDN)                         | Yes, we host the recordings and provide you with CDN URL directly through our Recording Assets API. | No, you are required to set up your own CDN servers to allow on-demand playback. |
+
+> **Important Note - Zipped Recording Files**
+>
+> - For all the templates created before 27th August 2023, the M3U8 file is zipped and uploaded, irrespective of the storage you've configured. 
+> - For older templates with recordings hosted on 100ms, you have an option to switch from Zipped to Unzipped recording files upload, but you cannot switch back once you've saved that setting.
+> - If you've hosted the recordings in your personal cloud storage bucket, you will still have the option to choose between zipped or unzipped M3U8 file uploads.
+> - If you're using 100ms storage, you cannot upload utilise the option of zipped M3U8 upload on any new templates.
+
+
+### Fetching VOD recordings
+There are effectively 3 methods to access and consume the recordings.
+
+<Tabs id="fetching-vod" items={['Recording Assets API','Webhooks','Dashboard']} />
+    <Tab id='fetching-vod-0'>
+    
+    - All files generated after recording are called Recording Assets. These will include various video and stream recordings, chat, transcripts and summary. You can access these using the [Recording Assets API](/server-side/v2/api-reference/recording-assets/overview).
+    - VOD Recording is generated as an asset type called `room-vod`. Depending upon your choice of zipped or unzipped uploads, you can access the path using the [GET recording asset API](/server-side/v2/api-reference/recording-assets/get-asset).
+    - If you've configured 100ms as your cloud storage, you can also use [GET pre-signed URL API](/server-side/v2/api-reference/recording-assets/get-presigned-url) to get the CDN URL for on-demand playback by inputing it in a compatible player.
+    - If you've configured your own cloud bucket storage, the GET pre-signed URL will return the presigned url of M3U8 file. This file is not usable as it is not playable unless you connect your cloud bucket to a CDN to enable on-demand playback.
```

Comment:
`the GET pre-signed URL will return the presigned url of M3U8 file` - For customer's buckets, we can get presigned url only if the credentials the have provided to us have read permission of the resource. So we cannot always guarantee that presigned url would be returned. Just letting you know if you want to mention it.

Reply:
I think we can skip this point. This will be too much info. But I'll note this down anyway to mention this in the other docs.